src/main/io/ledstrip.c

```diff
@@ -280,13 +282,14 @@ void reevalulateLedConfig(void)
 }
 
 // get specialColor by index
-static hsvColor_t *getSC(ledSpecialColorIds_e index)
+static hsvColor_t* getSC(ledSpecialColorIds_e index)
 {
-    return colors(specialColors(0)->color[index]);
+    return colors(specialColors_System.color[index]);
```

Comment:
`specialColors()->color[index]`. _System sufix should not be externally visible (but it's hard to do in C)